Source/ADSRComponent.cpp

```diff
@@ -0,0 +1,140 @@
+/*
+  ==============================================================================
+
+    ADSRComponent.cpp
+    Created: 15 May 2021 1:13:56pm
+    Author:  Robin Su
+
+  ==============================================================================
+*/
+
+#include "ADSRComponent.h"
+
+ADSRWheel::ADSRWheel() {
+    setRotaryStyle();
+    addAndMakeVisible(&rotary);
+    addAndMakeVisible(&rotaryLabel);
+    
+    rotaryLabel.setColour(1, juce::Colours::white);
```

Comment:
We can also configure color at a high level. Any component a lookAndFeel is applied to also applies to it's children. Just food for thought, see https://docs.juce.com/master/tutorial_look_and_feel_customisation.html

Reply:
oh good catch - that was left over from when I was troubleshooting the component + label not appearing